src/main/resources/proto/confio/proofs.proto

```diff
@@ -0,0 +1,235 @@
+syntax = "proto3";
```

Comment:
IMHO, it would be good to not commit these 3rd-party `*.proto` files, as `panacea-core` and `panacea-js` do. (The single source of truth philosophy)

Then, developers are responsible for running `protodep up ..` manually before running `gradle protobuf`. 
(This mechanism is similar as `git submodule`.)